### PR TITLE
CUDA_Runtime: Restore v10.2

### PR DESCRIPTION
**Pull request by stemann**
Follow-up for #7623

### CODE REVIEW

File: C/CUDA/CUDA_Runtime/build_tarballs.jl [hunk 1]
```diff
@@ -157,11 +157,19 @@ for version in CUDA.cuda_full_versions
...
 
         should_build_platform(triplet(augmented_platform)) || continue
-        push!(builds,
-              (; dependencies=[Dependency("CUDA_Driver_jll"; compat="0.7")],
-                 script, platforms=[augmented_platform], products=get_products(platform),
-                 sources=get_sources("cuda", components; version, platform)
-        ))
+        if Base.thisminor(version) == v"10.2"
+            push!(builds,
+                (; dependencies=get_dependencies(),
```
Comment (stemann) [inline C/CUDA/CUDA_Runtime/build_tarballs.jl:162]: @maleadt Should get_dependencies include CUDA_Driver_jll for CUDA 10.2?

**Comment by stemann** [comment]
#7623 also provided artifacts for x86_64-apple-darwin - this platform is currently not included in this modification of CUDA_Runtime...

File: C/CUDA/CUDA_Runtime/build_tarballs.jl [hunk 2]
```diff
@@ -7,7 +7,7 @@ include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
 include(joinpath(YGGDRASIL_DIR, "platforms", "cuda.jl"))
 
 name = "CUDA_Runtime"
-version = v"0.10.1"
+version = v"0.10.2"
```
Comment (maleadt) [inline C/CUDA/CUDA_Runtime/build_tarballs.jl:10]: Make this `0.11`; too many things changed for this to be an automatic update.

File: C/CUDA/CUDA_Runtime/build_tarballs.jl (hunk 1 above)
Comment (stemann) [reply C/CUDA/CUDA_Runtime/build_tarballs.jl:162]: I guess I was just trying to minimize the "amount of 10.2" that were getting into the clean, 11+ files...
 
And CUDA_Driver_jll is ready for 10.2, right?

File: C/CUDA/CUDA_Runtime/build_tarballs.jl (hunk 2 above)
Comment (stemann) [reply C/CUDA/CUDA_Runtime/build_tarballs.jl]: Done

File: C/CUDA/CUDA_Runtime/build_tarballs.jl (hunk 1 above)
Comment (maleadt) [reply C/CUDA/CUDA_Runtime/build_tarballs.jl]: CUDA_Driver_jll is just a package that attempts to load a more recent driver, and if it does, the 10.2 runtime won't get used. Doesn't matter for the devices you're targeting, though.

Comment (stemann) [reply C/CUDA/CUDA_Runtime/build_tarballs.jl]: OK. I have removed the `get_dependencies` function, and added CUDA_Driver_jll as a dep.

**Pull request merged**